docs/source/getting-started/networking-basics.md

```diff
@@ -25,8 +25,7 @@ Or by placing the following lines in a **configuration file**,
 `jupyterhub_config.py`:
 
 ```python
-c.JupyterHub.ip = '192.168.1.2'
-c.JupyterHub.port = 443
+c.JupyterHub.bind_ip = '192.168.1.2:443'
```

Comment:
it's `bind_url` here:

```suggestion
c.JupyterHub.bind_url = 'https://192.168.1.2:443'
```

Reply:
agree with @minrk but probably understood that http:// or https:// based on no-ssl or ssl used.